### PR TITLE
Update Dependencies section

### PR DESCRIPTION
**Pull request by nplakhtii**
Update Dependencies table
Change Minimal version to Recommended version

Relates-To: OLPEDGE-2794

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The table below lists the platforms on which the Data SDK has been tested.
 
 The table below lists the dependencies of the Data SDK.
 
-| Library              | Minimum version |
+| Library              | Recommended version |
 | :------------------- | :-------------- |
```
Comment (andrey-kashcheev) [inline README.md:68]: Should the whole table be re-aligned now?

Comment (nplakhtii) [reply README.md:68]: It renders well. However, I can improve it.

File: README.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ The table below lists the platforms on which the Data SDK has been tested.
...
 | libcurl              | 7.52.0          |
 | OpenSSL              | 1.1.1           |
```
[the remark this answers is not in the document]
Comment (nplakhtii) [reply README.md]: and also done. 

Aligned table borders and added "g" to version number 1.1.1

[the remark this answers is not in the document]
Comment (nplakhtii) [reply README.md]: @andrey-kashcheev, could you please have a look

**Review by andrey-kashcheev — APPROVED** [review]

**Review by HalynaDumych — APPROVED** [review]

**Pull request merged**